README.md

```diff
@@ -3,6 +3,20 @@
 
 # MozDef: Mozilla Enterprise Defense Platform
 
+## Documentation:
+
+http://mozdef.readthedocs.org/en/latest/
+
+
+## Give MozDef a Try in AWS:
+
+The following button will launch the Mozilla Enterprise Defense Platform in your AWS account.
+
+**Click with Caution:** Pressing the "Launch Stack" button and following through with the deployment will incur charges to your AWS account. 
```

Comment:
Can we change this language to align with "Warning:" vs Click with Caution or simple "Note:"